src/components/Sketches/components/EditSketchModal.js

```diff
@@ -224,24 +219,26 @@ class EditSketchModal extends React.Component {
               <Button
                 color="primary"
                 size="lg"
-                onClick={() => {
-                  this.setState({ onThumbnails: true });
-                }}
+                onClick={() => setOnThumbnails(true)}
+                onKeyDown={() => {}}
```

Comment:
The keydown function should be equivalent to the click function; the purpose is to aid those who can’t use a mouse